src/AutosuggestContainer.js

```diff
@@ -69,13 +69,14 @@ export default class AutosuggestContainer extends Component {
     renderSectionTitle: PropTypes.func,
     getSectionSuggestions: PropTypes.func,
     focusInputOnSuggestionClick: PropTypes.bool,
+    selectFirstSuggestion: PropTypes.bool,
     theme: PropTypes.object,
     id: PropTypes.string
   };
 
   static defaultProps = {
     onSuggestionsUpdateRequested: noop,
-    shouldRenderSuggestions: value => value.trim().length > 0,
+    shouldRenderSuggestions: value => value && value.trim().length > 0,
```

Comment:
Makes sense! How about we make sure that we never pass `null` to `shouldRenderSuggestions()` instead?
You fixed the problem for those who don't define their own `shouldRenderSuggestions()`, but those who do define their own will still get `null`, right?